livingdoc-converters/src/test/kotlin/org/livingdoc/converters/collection/CollectionConverterContract.kt

```diff
@@ -0,0 +1,15 @@
+package org.livingdoc.converters.collection
+
+import java.lang.reflect.AnnotatedElement
+import kotlin.reflect.KClass
+
+internal abstract class CollectionConverterContract<T: Collection<Any>> {
```

Comment:
The contract should include the common tests of its subclasses. Currently it defines no tests at all while the subclasses have some common tests.